scala/sag_et.par

```diff
@@ -4,9 +4,11 @@
 ! s = list of subdivisions for above division using non-native fifth
 !
 n 5 3
+|\
 n 7 4
+|\  ||\

```

Comment:
You need to replace "||" with "N" everywhere (double-shafts).

Reply:
Yes! Nice catch. Thank you.